packages/web/components/Player/PlayerAvatar.tsx

```diff
@@ -0,0 +1,24 @@
+import { Avatar, AvatarProps } from '@metafam/ds';
+import React from 'react';
+
+import { PlayerFragmentFragment } from '../../graphql/autogen/types';
```

Comment:
these imports can also be simplified by using absolute paths.
```suggestion
import { PlayerFragmentFragment } from 'graphql/autogen/types';
```